### ISSUE TITLE
why use 1 bit to store 0

### ISSUE THREAD

**Issue by DAVIDNEWGATE** [Zhouaojun/Incremental-Network-Quantization#12]
For example, for 4bit why don't we just use 15 to form quantized region, and another one such as 0000 to represent 0. Will unbalanced region influence INQ result?

**Comment by AojunZhou** [comment]
yes, for 4 bits, you can use 15 no-zero values + 0, unbalanced quantization doesn't influence INQ result, specifically, the INQ is very flexible，you can adjust bit-width according your task, for example , you can throw away 0，then, the 5 bits in INQ paper can be replaced by 4 bits.

**Comment by KangolHsu** [comment]
1.  template double weightCluster<int>(int weight,int M);
2.  template double weightCluster_zero<float>(float weight,int M);
why this two method is all the same？
the second one shoud contain 0？ @Zhouaojun
